build.sbt

```diff
@@ -0,0 +1,20 @@
+lazy val root = (project in file("."))
+  .enablePlugins(ScriptedPlugin)
+  .settings(
+    name := "scala-native.g8",
+    Test / test := {
+      val _ = (Test / g8Test).toTask("").value
+    },
+    scriptedLaunchOpts ++= List(
+      "-Xms1024m",
+      "-Xmx1024m",
+      "-XX:ReservedCodeCacheSize=128m",
+      "-Xss2m",
+      "-Dfile.encoding=UTF-8"
+    ),
+    scriptedBufferLog := false,
+    resolvers += Resolver.url(
+      "typesafe",
+      url("https://repo.typesafe.com/typesafe/ivy-releases/")
+    )(Resolver.ivyStylePatterns)
```

Comment:
I'm pretty sure this is not needed.

Reply:
I removed the resolver and updated to sbt `1.3.13` so it should be good to go now.